packages/main/src/SliderBase.ts

```diff
@@ -180,12 +189,16 @@ abstract class SliderBase extends UI5Element {
 
 	_handleMove(e: TouchEvent | MouseEvent) {} // eslint-disable-line
 
-	_handleUp() {}
+	_handleUp(e: TouchEvent | MouseEvent) {}	// eslint-disable-line
 
 	_onmousedown(e: TouchEvent | MouseEvent) {} // eslint-disable-line
 
+	_updateValueFromInput(e: KeyboardEvent | FocusEvent) {} // eslint-disable-line
+
 	_handleActionKeyPress(e: Event) {} // eslint-disable-line
 
+	_updateInputValue() {} // eslint-disable-line
```

Comment:
`eslint-disable-line`

Reply:
This function is implemented differently in both Slider.ts and RangeSlider.ts but the typescript wants to be defined in their super class also.